component/prometheus/integration/integrationComponent.go

```diff
@@ -0,0 +1,119 @@
+package integration
```

Comment:
Could this code also be used for non prometheus integrations (eventhandler)? also nit: use `_` or `-` for a filename instead of camel case?

Reply:
Yes, it possibly could. We may need to add additional extension points for that, but it could be done.